### PR TITLE
feat(cli): Add a output for written url to argo workflow output.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: packages/cli/src/cli/config/action.imagery.config.ts [hunk 1]
```diff
@@ -120,13 +128,12 @@ export class CommandImageryConfig extends CommandLineAction {
...
-      logger.info(
-        { path: output, url: `https://basemaps.linz.govt.nz/?config=${configPath}&tileMatrix=NZTM2000Quad${location}` },
-        'ImageryConfig:Done',
-      );
+      const outputPath = fsa.join(path, `basemaps-config-${configJson.hash}.json.gz`);
+      await fsa.writeJson(outputPath, configJson);
+      const configPath = base58.encode(Buffer.from(outputPath));
+      const url = `https://basemaps.linz.govt.nz/?config=${configPath}&i=${name}&tileMatrix=NZTM2000Quad${location}`;
+      logger.info({ path: output, url }, 'ImageryConfig:Done');
+      if (output != null) fsa.write(output, url);
```
Comment (blacha) [inline packages/cli/src/cli/config/action.imagery.config.ts:136]: you need to `await` this

Comment (Wentao-Kuang) [reply packages/cli/src/cli/config/action.imagery.config.ts]: Good pick. Fixed.

Comment (blacha) [reply packages/cli/src/cli/config/action.imagery.config.ts]: is there anyway in argo to just set a env var or do we always have to use files for outputs?

**Review by blacha — APPROVED** [review]

File: packages/cli/src/cli/config/action.imagery.config.ts (hunk 1 above)
Comment (Wentao-Kuang) [reply packages/cli/src/cli/config/action.imagery.config.ts]: We should investigate this later. Currently what I see is just parameter output and artifacts output.

**Pull request merged**